### PR TITLE
Feat/cms

### PR DESCRIPTION
**Pull request by juanfcarrillo**

### CODE REVIEW

File: .env_explample
Comment (ferjozsot23) [inline .env_explample:1]: Add EOL

File: src/pages/colegio.astro [hunk 1]
```diff
@@ -1,14 +1,51 @@
...
+			href="#school-form"
+			title="Contrita Campamento de Verano Espacial "
+			body={`¡La aventura te espera!\nInscríbete al Campamento Espacial más grande del país.`}
+			image={Rocket}
+		/>
+		<!-- <StoryblokComponent blok={content} /> -->
+		<Divider />
+		<CountDown
+			countdownDate="jul 11, 2024 10:00:00"
+		/>
```
Comment (ferjozsot23) [inline src/pages/colegio.astro:32]: Is this date modifiable in the cms? and there is a data validator or error handling?

File: src/components/organisms/CountDown.astro [hunk 2]
```diff
@@ -44,15 +50,15 @@ const t = useTranslations(lang);
   </div>
 </div>
 
-<script>
+<script define:vars={{ countdownDate }}>
   // countdown.js
   document.addEventListener("DOMContentLoaded", () => {
     // count-down timer
-    let dest = new Date("apr 25, 2024 10:00:00").getTime();
+    let dest = new Date(countdownDate).getTime();
```
Comment (ferjozsot23) [inline src/components/organisms/CountDown.astro:57]: What if date parser failed?

File: src/components/sections/CTAFull.astro [hunk 3]
```diff
@@ -5,15 +5,42 @@ interface Props {
     body?: string;
     button?: string;
     href?: string;
+    fullHeight?: boolean;
+    background?:
+        | {
+              from: string;
+              to: string;
+          }
```
Comment (ferjozsot23) [inline src/components/sections/CTAFull.astro:13]: In future, consider create a interface for readability types

File: src/components/sections/CTAFull.astro [hunk 4]
```diff
@@ -5,15 +5,42 @@ interface Props {
...
 
-const { image } = Astro.props;
+const { image, fullHeight, background, backgroundImage } = Astro.props;
+
+const backgroundColor =
+    typeof background === "string"
+        ? {
+              from: background,
+              to: background,
+          }
```
Comment (ferjozsot23) [inline src/components/sections/CTAFull.astro:27]: In future, consider create a interface for readability types

**Review by ferjozsot23 — CHANGES_REQUESTED** [review]
Great job!!! Just few questions and suggestions 🚀

File: src/components/organisms/CountDown.astro (hunk 2 above)
Comment (juanfcarrillo) [reply src/components/organisms/CountDown.astro:57]: The cms is responsible of the date validation

File: src/components/sections/CTAFull.astro (hunk 3 above)
Comment (juanfcarrillo) [reply src/components/sections/CTAFull.astro:13]: Ok i'll consider that

File: .env_explample
Comment (juanfcarrillo) [reply .env_explample:1]: There's no need of that right now because i'd had to create another branch for that 😅

**Comment by juanfcarrillo** [comment]
> Great job!!! Just few questions and suggestions 🚀

Please don't requests changes for questions because I'll have to modify the code in order to unblock the PR 😢 
Can you remove the request pls :3

**Comment by ferjozsot23** [comment]
> > Great job!!! Just few questions and suggestions 🚀
> 
> Please don't requests changes for questions because I'll have to modify the code in order to unblock the PR 😢

**Review by ferjozsot23 — APPROVED** [review]

File: src/pages/colegio.astro (hunk 1 above)
Comment (juanfcarrillo) [reply src/pages/colegio.astro:32]: Yes, right now we only want to test the cms with test-post route. And the cms is responsible for that validation

**Pull request merged**